src/TableData.cpp

```diff
@@ -151,6 +151,29 @@ K writeReadTable(K schema_id, K array_data, K options)
   KDB_EXCEPTION_CATCH;
 }
 
+arrow::Compression::type getCompressionType(std::string compression) {

```

Comment:
```suggestion
arrow::Compression::type getCompressionType(const std::string& compression) {

```

Reply:
BTW, there is actually an Arrow API just for this: https://arrow.apache.org/docs/cpp/api/utilities.html#_CPPv4N5arrow4util5Codec18GetCompressionTypeERKNSt6stringE